maintainers/maintainer-list.nix

```diff
@@ -10829,6 +10829,11 @@
     }];
     name = "Yaroslav Bolyukin";
   };
+  lachrymal = {
```

Comment:
It's better to add an email although there is no standard yet. It is possible that when you abandon this account, others will still want to communicate with you about changes or removals of maintainer information.